src/styles.ts

```diff
@@ -193,7 +193,10 @@ export const styles = css`
 		width: var(--width);
 		min-width: calc(2ch + 0.25em);
 	}
-	:host([type='color']) .line {
+	:host([type='color']) .line,
+	:host([type='checkbox']) .line {
```

Comment:
What is this for?

Reply:
That's for removing the line shown into an input component (e.g. a text one). Maybe it can be worked-around inserting a ```?hiddenLine``` attribute to be set according to the creating function?

Reply:
Nevermind, I set it in the scoped style of the component. I will take care of the error and warning support now